dbt/models/reporting/reporting.vw_top_5_muni.sql

```diff
@@ -63,8 +63,11 @@ most_recent_values AS (
         pin,
         year,
         oneyr_pri_board_tot AS prior_bor_av,
-        COALESCE(certified_tot, mailed_tot) AS ccao_av,
+        COALESCE(certified_tot, mailed_tot, pre_mailed_tot) AS ccao_av,
         CASE
+            WHEN
+                (certified_tot IS NULL AND mailed_tot IS NULL)
+                THEN 'pre-mailed'
```

Comment:
Only adding the pre-mailed provisional stage here to avoid null AVs for towns that haven't mailed yet, per the test that was failing. It wouldn't be too hard to add the pre-certified provisional stage too, but I'm avoiding it until we decide it's necessary.